mangaki/mangaki/static/js/vote.js

```diff
@@ -210,6 +210,10 @@ Slot.prototype.fetch = function () {
 /* Mosaic takes care of mapping Cards inside an element with Slots pointing to
  * some remote URLs on the server.
  */
+function buildSlotURL(category, slot_sort) {
```

Comment:
Can you keep the documentation for the `Mosaic` function close to the `Mosaic` function?